packages/aws-cdk-lib/aws-eks/lib/cluster.ts

```diff
@@ -2681,3 +2731,92 @@ function cpuArchForInstanceType(instanceType: ec2.InstanceType) {
 function flatten<A>(xss: A[][]): A[] {
   return Array.prototype.concat.call([], ...xss);
 }
+
+function validateRemoteNetworkConfig(props: ClusterProps) {
```

Comment:
IMO, this can be a private method in the class which is what typically CDK does for validation methods.

Reply:
Thank you for the pointer! Pushed an update.